src/components/SocialMediaLinks/SocialMediaLinks.test.tsx

```diff
@@ -0,0 +1,243 @@
+import { render, screen } from "@testing-library/react";
+import { axe } from "jest-axe";
+import * as React from "react";
+import renderer from "react-test-renderer";
+
+import SocialMediaLinks from "./SocialMediaLinks";
+
+// @todo If I try and use this as a linkData prop in any of the SocialMediaLinks components being rendered below, I get a type error.
```

Comment:
I see. Well I decided to drop the use of a variable anyway. But related: SocialMediaLinkDataProps is already exported as an interface. Should it be exported again, as a type, so that other devs can more easily assign data props to a variable? If yes, should SocialMediaLinksProps also be exported as a type?

Reply:
Yes to `SocialMediaLinkDataProps` but no to `SocialMediaLinkProps`. A consuming app will not generally not need to extend or implement all props in `SocialMediaLinkProps`. The `SocialMediaLinkDataProps` type is specific for the data structure of a prop so that could be used by an app.

You exported the type in the file but there's one final step to export the component and type for a consuming app. We can talk about that small update once this PR is ready for a review.

Reply:
I left SocialMediaLinkDataProps as an interface, as a result of [a different conversation regarding duplication](https://github.com/NYPL/nypl-design-system/pull/1391/files#r1319053460). I am unsure if that is what we desire, ultimately.